components/Balance.tsx

```diff
@@ -0,0 +1,55 @@
+import Image from 'next/image';
+import EnoughToBuy from './EnoughToBuy';
+
+type Props = {
+  glo: number;
+  setGlo: React.Dispatch<React.SetStateAction<number>>;
+  yearlyYield: number;
+};
+
+export default function Balance({ glo, setGlo, yearlyYield }: Props) {
+  const formattedGlo = new Intl.NumberFormat("en-US", {
+    style: "currency",
+    currency: "USD",
+    minimumFractionDigits: 2,
+    maximumFractionDigits: 2,
+  }).format(glo);
+  const formattedYearlyYield = new Intl.NumberFormat("en-US", {
+    minimumFractionDigits: 0,
+    maximumFractionDigits: 1,
+  }).format(yearlyYield);
+
+  const splitFmtGlo = formattedGlo.split(".");
+  const fmtGloDollarPart = splitFmtGlo[0];
+  const fmtGloCentPart = splitFmtGlo[1];
+
+  return (
+    <div className="bg-white rounded-[20px] pt-4">
+      <div className="flex flex-col space-y-2 p-4">
+      <div className="self-center text-[1.1rem] text-pine-700/90">Balance</div>
+        <div className="flex flex-row font-semibold justify-center">
+          <div className="flex flex-row text-[2.625rem] items-baseline">
```

Comment:
That's a lot of flex :P

Reply:
do you want a separate component library?

Reply:
who reverted my grid layout last time? @har777 ?